crates/nu-cmd-extra/src/extra/bits/and.rs

```diff
@@ -42,52 +58,82 @@ impl Command for BitsAnd {
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let head = call.head;
-        let target: i64 = call.req(engine_state, stack, 0)?;
+        let target: Value = call.req(engine_state, stack, 0)?;
+        let endian = call.get_flag::<Value>(engine_state, stack, "endian")?;
+
+        let little_endian = match endian {
+            Some(val) => {
+                let span = val.span();
+                match val {
+                    Value::String { val, .. } => match val.as_str() {
+                        "native" => cfg!(target_endian = "little"),
+                        "little" => true,
+                        "big" => false,
+                        _ => {
+                            return Err(ShellError::TypeMismatch {
+                                err_message: "Endian must be one of native, little, big"
+                                    .to_string(),
+                                span,
+                            })
+                        }
+                    },
+                    _ => false,
+                }
+            }
+            None => cfg!(target_endian = "little"),
+        };
 
-        // This doesn't match explicit nulls
```

Comment:
I'm not sure if this comment can be removed

Reply:
removed this when I changed the implementation of `and, or, xor` to use `operate` which made that section redundant, but since changing it back I forgot to add back the comment. fixing next commit